util/chplenv/__init__.py

```diff
@@ -28,6 +28,7 @@
     'chpl_3p_qthreads_configs',
     'chpl_3p_re2_configs',
     # General purpose helpers
+    'chpl_python_version',
```

Comment:
should be alphabetize within the sublist. i.e. should come after chpl_home_utils.